deployment/docker/nginx.conf

```diff
@@ -47,6 +47,22 @@ http {
         server_name _;
         index index.php index.html;
         root /var/www;
+        
+        add_header 'Access-Control-Allow-Origin' 'https://checkin.eventyay.com' always;
```

Comment:
Please change this into a variable. So we can use different URLs, e.g. for the development branch.

Reply:
How about using wildcard `*` to allow all "origin"?

Reply:
Well, for partners such as Wikimedia we also want to deploy it to different domains.